### PR TITLE
Feature/eventpage

### PR DESCRIPTION
**Pull request by notBigO**

### CODE REVIEW

File: src/app/page.js [hunk 1]
```diff
@@ -1,113 +1,53 @@
...
-            Docs{' '}
-            <span className="inline-block transition-transform group-hover:translate-x-1 motion-reduce:transform-none">
-              -&gt;
-            </span>
-          </h2>
-          <p className={`m-0 max-w-[30ch] text-sm opacity-50`}>
-            Find in-depth information about Next.js features and API.
-          </p>
-        </a>
+const pexel = (id) =>
```
Comment (sid-js) [inline src/app/page.js]: Can you please abstract this away from the page file. Either put it in component, or a json file and import it.

File: src/app/page.js [hunk 2]
```diff
@@ -1,113 +1,53 @@
...
+  `https://images.pexels.com/photos/${id}/pexels-photo-${id}.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260`;
+const images = [
```
Comment (sid-js) [inline src/app/page.js]: Abstract this away to somewhere else

**Review by sid-js — CHANGES_REQUESTED** [review]
Please do the required small changes

**Review by notBigO — COMMENTED** [review]
added requested fix and frame focus working now too

**Review by notBigO — COMMENTED** [review]
refactored code

**Pull request merged**